CONTRIBUTING.md

```diff
@@ -46,6 +46,17 @@ There are a few ways to do this, but we prefer to use
 create and manage a Python virtual environment specific to this
 project.
 
+If you already have `pyenv` and `pyenv-virtualenv` configured you can
+take advantage of the `setup-env` tool in this repo to automate the
+entire environment configuration process.
+
+```bash
```

Comment:
I believe this should be `console`, not `bash`.

Reply:
I'll reserve a CVE for it Dr.  
🙄 🤣

Reply:
See: 3c2781c